### PR TITLE
[Backport 2.x] [Spotless] Applying Google Code Format for core/src/main files #2 #1931

### PR DESCRIPTION
**Pull request by MitchellGale**
### Description

This PR applies Spotless on Java code for:
`core/src/main/java/org/opensearch/sql/exception`
`core/src/main/java/org/opensearch/sql/executor`
`core/src/main/java/org/opensearch/sql/expression` 
`core/src/main/java/org/opensearch/sql/monitor`

 
### Issues Resolved
[[List any issues this PR will resolve]](https://github.com/opensearch-project/sql/issues/1101)
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [x] New functionality has javadoc added
  - [x] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by Yury-Fridlyand** [comment]
Please, disable checkstyle and rerun spotless

**Review by acarbonetto — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -217,5 +219,12 @@ task updateVersion {
         println "Setting version to ${newVersion}."
         // String tokenization to support -SNAPSHOT
         ant.replaceregexp(file:'build.gradle', match: '"opensearch.version", "\\d.*"', replace: '"opensearch.version", "' + newVersion.tokenize('-')[0] + '-SNAPSHOT"', flags:'g', byline:true)
+        ant.replaceregexp(match:'"version": "\\d+.\\d+.\\d+.\\d+', replace:'"version": ' + '"' + newVersion.tokenize('-')[0] + '.0', flags:'g', byline:true) {
```
Comment (Yury-Fridlyand) [inline build.gradle:222]: revert

File: build.gradle [hunk 2]
```diff
@@ -113,9 +116,8 @@ allprojects {
         sourceCompatibility = targetCompatibility = "11"
     }
     configurations.all {
-        resolutionStrategy.force "com.squareup.okio:okio:3.5.0"
-        resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib:1.9.0"
-        resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.9.0"
+        resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib:1.6.0"
```
Comment (Yury-Fridlyand) [inline build.gradle:119]: revert

File: build.gradle (hunk 1 above)
Comment (MitchellGale) [reply build.gradle]: reverted

File: build.gradle (hunk 2 above)
Comment (MitchellGale) [reply build.gradle]: reverted

**Review by Yury-Fridlyand — APPROVED** [review]

**Review by acarbonetto — APPROVED** [review]

**Review by matthewryanwells — APPROVED** [review]

**Review by Yury-Fridlyand — DISMISSED** [review]

**Review by acarbonetto — DISMISSED** [review]

**Review by acarbonetto — APPROVED** [review]

**Review by Yury-Fridlyand — APPROVED** [review]

**Pull request merged**